### PR TITLE
Add option.update action and tests

### PR DESCRIPTION
**Pull request by reiterl**

### CODE REVIEW

File: openslides_backend/action/actions/option/update.py [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+from ...util.default_schema import DefaultSchema
+from ...util.register import register_action
+from ..vote.create import VoteCreate
+from ..vote.update import VoteUpdate
+
+
+@register_action("option.update")
+class OptionUpdateAction(UpdateAction):
+    """
+    Action to update a user.
```
Comment (jsangmeister) [inline openslides_backend/action/actions/option/update.py:17]: c&p typo

File: openslides_backend/action/actions/option/update.py [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+    """
+
+    model = Option()
+    schema = DefaultSchema(Option()).get_update_schema(
+        additional_optional_fields={
+            "Y": {"type": "string", "pattern": r"^-?(\d|[1-9]\d+)\.\d{6}$"},
+            "N": {"type": "string", "pattern": r"^-?(\d|[1-9]\d+)\.\d{6}$"},
+            "A": {"type": "string", "pattern": r"^-?(\d|[1-9]\d+)\.\d{6}$"},
```
Comment (jsangmeister) [inline openslides_backend/action/actions/option/update.py:25]: Please use the decimal schema here that you created in another PR.

File: openslides_backend/action/actions/option/update.py [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+from ....shared.exceptions import ActionException
+from ....shared.patterns import Collection, FullQualifiedId
+from ...generics.update import UpdateAction
+from ...util.default_schema import DefaultSchema
+from ...util.register import register_action
+from ..vote.create import VoteCreate
+from ..vote.update import VoteUpdate
+
+
+@register_action("option.update")
```
Comment (jsangmeister) [inline openslides_backend/action/actions/option/update.py:14]: I guess this action should also be internal? I don't see why the client would or should ever set the values on an option directly

Comment (jsangmeister) [reply openslides_backend/action/actions/option/update.py:14]: If that's the case, you can probably fuse this with the `set_auto_fields` action from the other PR. We don't need 2 internal actions on the same fields.

File: openslides_backend/action/actions/option/update.py [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+                payload_create.append(
+                    {
+                        "option_id": instance["id"],
+                        "value": "A",
+                        "weight": instance["abstain"],
+                        "meeting_id": option["meeting_id"],
+                    }
+                )
+            else:
+                payload_update.append({"id": vote_id, "weight": instance["abstain"]})
```
Comment (jsangmeister) [inline openslides_backend/action/actions/option/update.py:80]: You can reduce the amount of redundant code by iterating over the needed values (`yes`, `no`, `abstain`)

File: openslides_backend/action/actions/vote/create.py [hunk 5]
```diff
@@ -0,0 +1,17 @@
+from ....models.models import Vote
+from ...generics.create import CreateAction
+from ...util.default_schema import DefaultSchema
+from ...util.register import register_action
+
+
+# TODO should be internal
```
Comment (jsangmeister) [inline openslides_backend/action/actions/vote/create.py:7]: Please set it to internal then

**Review by jsangmeister — CHANGES_REQUESTED** [review]
I do not see the need for this action, please explain it to me. Also, basic tests for `vote.create` and `vote.update` are missing.

**Comment by reiterl** [comment]
This action should be used in the analog type to update the options directly.
The poll.vote action should not be used for for update in analog case.

File: openslides_backend/action/actions/option/update.py (hunk 1 above)
Comment (reiterl) [reply openslides_backend/action/actions/option/update.py:17]: Relabeled.

File: openslides_backend/action/actions/option/update.py (hunk 4 above)
Comment (reiterl) [reply openslides_backend/action/actions/option/update.py:80]: Use a loop to simplify the code here.

File: openslides_backend/action/actions/vote/create.py (hunk 5 above)
Comment (reiterl) [reply openslides_backend/action/actions/vote/create.py:7]: Set to internal.

File: openslides_backend/action/actions/option/update.py (hunk 3 above)
Comment (reiterl) [reply openslides_backend/action/actions/option/update.py:14]: In the analog type we want to update options directly. So it is not internal.
The set_auto_fields does something different then this action.

File: openslides_backend/action/actions/option/update.py (hunk 2 above)
Comment (reiterl) [reply openslides_backend/action/actions/option/update.py:25]: Include the decimal schema in schema and use it here.

**Comment by reiterl** [comment]
Add a tests for vote create and vote update.

**Review by jsangmeister — CHANGES_REQUESTED** [review]
Test for `vote.create` is still missing...

**Review by jsangmeister — APPROVED** [review]
My bad, `vote.create` is already tested. Only rebase needed

**Pull request merged**